src/huntsman/pocs/dome/musca.py

```diff
@@ -104,34 +109,28 @@ def door_closed(self):
         return v == Protocol.DOOR_CLOSED
 
     @property
-    def status(self):
-        """A dictionary containing all status info for dome.
-
-        TODO: Add other info (e.g. dome moving)
-        TODO: Auto-update this every ~60s
-        """
-        if self._status_timer is None:
-            update_status = True
-        else:
-            update_status = self._status_timer.expired()
+    def is_safe_to_open(self):
+        v = self.status[Protocol.BATTERY]
+        if v < self.MIN_OPERATING_VOLTAGE:
+            self.logger.debug('Dome shutter battery voltage too low to open: {!r}', v)
+            return False
+        return True
 
-        if update_status:
+    @property
+    def status(self):
+        """A dictionary containing all status info for dome. """
+        with self._command_lock:  # Make status call thread-safe
             for i in range(1, self._max_status_attempts + 1):
+                try:
+                    status = self._get_status_dict()
+                    status["dome_thread_running"] = self._dome_thread.is_alive()
```

Comment:
This is the dome status, so `dome_thread_running` isn't informative to the user.

```suggestion
                    status["status_thread_running"] = self._dome_thread.is_alive()
```